### ISSUE TITLE
clang warning

### ISSUE THREAD

**Issue by polluks** [gwsw/less#421]
```
cc -Wall -O2   -c -o run.o run.c
run.c:207:6: warning: variable 'tests' set but not used [-Wunused-but-set-variable]
        int tests = 0;
            ^
1 warning generated.
```
Make hangs after this line somehow
```
cc -Wall -O2  -o lt_screen lt_screen.o unicode.o wchar.o

**Comment by gwsw** [comment]
Should be fixed in c56b8097e1a607587adc2be4c8ff195213cdd124.

**Comment by polluks** [comment]
thx
